src/thirdparty/linenoise.hpp

```diff
@@ -2404,6 +2430,13 @@ inline const std::vector<std::string>& GetHistory() {
     return history;
 }
 
+static ssize_t writeFixed(int fd, const void *buf, size_t n)
```

Comment:
This function does not appear to be used. Can we leave it out? If not, where should we use it?

Either way, macOS CI is complaining because this function isn't returning the `ssize_t` its signature promises.